src/main/java/com/autotune/analyzer/experiment/ExperimentResultValidation.java

```diff
@@ -64,7 +64,7 @@ public void validate(List<ExperimentResultData> experimentResultDataList, Map<St
 
                     // Check if there is only one K8sObject in resultData
                     if (resultData.getKubernetes_objects().size() > 1) {
-                        errorMsg = errorMsg.concat(String.format("Multiple Kubernetes Objects are detected for result data of experiment : {} .Multiple Kubernetes Objects are unsupported", expName));
+                        errorMsg = errorMsg.concat(String.format("Multiple Kubernetes Objects are detected for result data of experiment: %s .Multiple Kubernetes Objects are unsupported", expName));
```

Comment:
I think this message can be shortened to something like this:

`Bulk Kubernetes Objects are unsupported!`

It will be consistent with bulk experiment entries response.

Reply:
Thanks for the review @khansaad